exonum/src/node/mod.rs

```diff
@@ -314,6 +314,24 @@ impl NodeConfig<PathBuf> {
     }
 }
 
+impl<T> NodeConfig<T> {
+    fn validate_or_panic(&self) {
+        let capacity = &self.mempool.events_pool_capacity;
+        if capacity.internal_events_capacity == 0 {
```

Comment:
I'd say that `internal_events_capacity` should be more than zero. But there is a little problem here. `internal_events_capacity` has a `usize` type. So if you specify negative number the overflow occur and we'll get huge size of the capacity.

Reply:
And take into account this case in tests please.

Reply:
Added more sanity checks and a few tests.